config/locales/fr.yml

```diff
@@ -516,9 +516,9 @@ fr:
     already_ordered:
       cart: "panier"
       message_html: "Vous avez déjà passé une commande pour ce cycle de vente. Vérifiez votre %{cart} pour voir les produits commandés. Vous pouvez annuler ou modifier votre commande jusqu'à la fermeture du cycle de vente."
-  home:
+  shops:
```

Comment:
I thought about this a bit more and it is very unlikely to cause a conflict. But Transifex will overwrite `fr.yml` once it's translated. Just when we get new translations from Transifex and have changes in the master branch at the same position of the locale, we can get merge conflicts.